src/data-types.js

```diff
@@ -87,6 +87,18 @@ export function getTypes(options) {
 		}
 	}
 
+	var int64 = {
+		unpack: function(buffer){
+			var value = buffer.readUInt64LE(buffer.pointer);
```

Comment:
@IvanSanchez The Gobble build system wouldn't be able to validate if a function exists, though, because Javascript objects are dynamic.

Unit tests would be very handy, but wouldn't we need some way to run an Artemis server in a controlled environment, and then force it to send a certain packet so we can validate it?